webui/react/src/omnibar/tree-extension/trees/dev.ts

```diff
@@ -42,6 +50,7 @@ const dev: TreeNode[] = [
     title: 'resetLocalStorage',
   },
   {
+    closeBar: true,
```

Comment:
**question:** The `closeBar` property is working nicely. What's your approach in terms of figuring out what commands should/shouldn't close? Like I was trying out `dev serverAddress reset` and was expecting the bar to close upon success.

Reply:
hmm, these are open to suggestions. generally, I was thinking

- if this command succeeds am I likely to want to run a similar command immediately?

serveraddres reset seems like a good candidate to close the bar after

Reply:
I added closeBar to some more commands. happy to tune that somemore after.